scripts/release.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
+const path = require('path');
+const { exec } = require('shelljs');
+const YAML = require('yaml');
+
+function commitTemplate() {
+  exec(`git add template.tpl`);
+  const { code } = exec(`git commit -m "chore: release new version"`);
+  if (code !== 0) {
+    console.error('git commit failed!');
+    process.exit(code);
+  }
+  return exec(`git log -1 --pretty=%H`)
+    .toString()
+    .trim();
+}
+
+function updateMetadata(commitHash) {
+  const filePath = path.resolve('metadata.yaml');
+  const yaml = YAML.parse(fs.readFileSync(filePath).toString());
+  yaml.versions = [
+    {
+      sha: commitHash,
+      changeNotes: 'Update the template',
```

Comment:
Wdyt about getting the commit title here?

Reply:
wdyt https://github.com/algolia/search-insights-gtm/pull/1/commits/5844689c5f091936ac8e42f75d6a9c421d04c5ed ?